OpenRA.Game/Map/MapPreview.cs

```diff
@@ -323,7 +323,8 @@ public void Install()
 						Game.RunAfterTick(() =>
 						{
 							var package = modData.ModFiles.OpenPackage(mapFilename, mapInstallPackage);
-							UpdateFromMap(package, mapInstallPackage, MapClassification.User, null, GridType);
+							if (package != null)
+								UpdateFromMap(package, mapInstallPackage, MapClassification.User, null, GridType);
```

Comment:
Possibly. Either way, `OpenPackage` returns null as a last resort. I'm fine with saying 'this will never happen' and ignoring the issue.